# Model
model: "sawyer_push.xml"

# Simulation
tf: 0.10	# [s] final time
tc: 5e-2 	# [s] control sampling period

# Contact pairs
## number of contact pairs
npair: 1
## indices of the sites on the robot
spair1: [4]
## indices of the corresponding sites in the environment	
spair2: [0]

# Task
## desired final pose of the control joint
desiredFinalPos: [1.3, 0, 0.0647, 0.0, 0.0, 0.0]
## desired final velocity of the control and robot joints
desiredFinalVel: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
## initial index of the DOF associated with the control joint
controlJointDOF0: 0

# Weights
w1: 1e4 	# weight on deviations in x and y directions
w2: 1e2  	# weight on deviations in z and orientation
w3: 2e0  	# weight on final velocities
w4: 2e0  	# weight on virtual stiffness
